ostelco-ios-client/Protocols/ApplePayDelegate.swift

```diff
@@ -38,6 +38,7 @@ extension ApplePayError: LocalizedError {
     }
 }
 
+// swiftlint:disable:next class_delegate_protocol
```

Comment:
Blergh, that's probably a swift 5 bug on swiftLint - now that you can specify that anything can be a protocol constraint (as UIViewController is here), SwiftLint keeps looking for the `ApplePayDelegate: class` declaration even though it's no longer necessary.